build.gradle

```diff
@@ -16,8 +16,120 @@
+
+allprojects  {
+    group = 'org.apache.fineract'
+
     repositories {
         jcenter()
     }
+
+    apply plugin: 'io.spring.dependency-management'
+    apply plugin: "com.diffplug.spotless"
+
+    // Configuration for the spotless plugin
+    // https://github.com/diffplug/spotless/tree/main/plugin-gradle
+    spotless {
+        format 'misc', {
+            target '**/*.md', '**/*.properties', '**/.gitignore', '**/*.yml', '**/*.xml', '**/**.json', '**/*.sql'
+            targetExclude '**/build/**', '**/bin/**', '**/.settings/**', '**/.idea/**', '**/.gradle/**'
+            indentWithSpaces(4)
+            endWithNewline()
+            trimTrailingWhitespace()
+        }
+
+        groovyGradle {
+            target '*.gradle', '**/*.gradle'
+            targetExclude '**/build/**'
+            greclipse()
+            indentWithSpaces(4)
+            endWithNewline()
+            trimTrailingWhitespace()
+        }
+
+        if (plugins.hasPlugin('java')) {
+            java {
+                targetExclude '**/build/**', '**/bin/**', '**/out/**'
+                importOrder() //sort imports alphabetically
+                removeUnusedImports()
+                eclipse().configFile "$rootDir/config/fineractdev-formatter.xml"
+                endWithNewline()
+                trimTrailingWhitespace()
+
+                // Enforce style modifier order
+                custom 'Modifier ordering', {
+                    def modifierRanking = [
+                        public      : 1,
+                        protected   : 2,
+                        private     : 3,
+                        abstract    : 4,
+                        default     : 5,
+                        static      : 6,
+                        final       : 7,
+                        transient   : 8,
+                        volatile    : 9,
+                        synchronized: 10,
+                        native      : 11,
+                        strictfp    : 12]
+                    // Find any instance of multiple modifiers. Lead with a non-word character to avoid
+                    // accidental matching against for instance, "an alternative default value"
+                    it.replaceAll(/\W(?:public |protected |private |abstract |default |static |final |transient |volatile |synchronized |native |strictfp ){2,}/, {
+                        // Do not replace the leading non-word character.  Identify the modifiers
+                        it.replaceAll(/(?:public |protected |private |abstract |default |static |final |transient |volatile |synchronized |native |strictfp ){2,}/, {
+                            // Sort the modifiers according to the ranking above
+                            it.split().sort({ modifierRanking[it] }).join(' ') + ' '
+                        }
+                        )
+                    }
+                    )
+                }
+            }
+
+            if (project.hasProperty("automatedBuild")) {
```

Comment:
This seems to be moving from another section? More out of curiosity, what is this for? /Cc @thesmallstar (FYI we are doing a massive "move", see FINERACT-1188 and FINERACT-1171 for more background)